packages/actions/types/Actions.d.ts

```diff
@@ -231,6 +230,15 @@ declare module 'grafx-studio-actions' {
          */
         export type LayoutWithMethods = Layout & LayoutMethods;
 
+        /**
+         * Represents a Date inside Actions
+         */
+        export interface DateValue {
```

Comment:
Date is in the js API (and in quickjs) so _overriding_ was not the best thing as it triggers wrong autocompletion...

other candidates:
- `VariableDate` (a bit confusing with `DateVariable`)
- `ValueDate` but we already have the `.dateValue` getter

Reply:
This is now using the native `Date` js type in the actions.